### PR TITLE
Update date, time and datetime widget render method to handle derived instance

### PR DESCRIPTION
**Pull request by Sir-heed**
**Problem**
[1894](https://github.com/django-import-export/django-import-export/issues/1894)

**Solution**
Replace the `type` check in the render method with `isinstance`

**Acceptance Criteria**
Added tests.
Ran existing test suite.

### CODE REVIEW

File: tests/core/tests/test_widgets.py [hunk 1]
```diff
@@ -117,6 +123,10 @@ def setUp(self):
     def test_render(self):
         self.assertEqual(self.widget.render(self.date), "13.08.2012")
 
+    def test_render_dervied_date(self):
```
Comment (matthewhegarty) [inline tests/core/tests/test_widgets.py:126]: typo in this method name

**Review by matthewhegarty — APPROVED** [review]
looks good, thanks for submitting - you have a couple of typos in method names.

File: tests/core/tests/test_widgets.py (hunk 1 above)
Comment (Sir-heed) [reply tests/core/tests/test_widgets.py]: Thanks, fixed.

**Review by matthewhegarty — APPROVED** [review]
Thanks - feel free to add your name to AUTHORS, and you can update Release Notes as well if you want.

**Pull request merged**